mne_icalabel/features/topomap.py

```diff
@@ -0,0 +1,141 @@
+from typing import Optional, Union
+
+import numpy as np
+from mne.channels.layout import _find_topomap_coords
+from mne.defaults import _BORDER_DEFAULT, _EXTRAPOLATE_DEFAULT
+
+try:
+    from mne.defaults import _INTERPOLATION_DEFAULT
+except ImportError:  # import is valid only for MNE ≥ 1.1
+    _INTERPOLATION_DEFAULT = "cubic"
+
+from mne.io import Info
+from mne.io.pick import (
+    _get_channel_types,
+    _pick_data_channels,
+    _picks_to_idx,
+    pick_info,
+)
+from mne.preprocessing import ICA
+from mne.utils import check_version
+from mne.viz.topomap import _check_extrapolate, _make_head_outlines, _setup_interp
+from numpy.typing import NDArray
+
+from ..utils._docs import fill_doc
+
+
+@fill_doc
+def get_topomaps(
```

Comment:
It seems like `get_topomaps` works on ICA instance, whereas `get_topomap` works on a np.ndarray. Perhaps in line with the MNE tfr module naming, we can make it more explicit like:

`get_topomaps` stays the same
`get_topomap` renamed to something like `get_topomap_array` to make it explicit that it works on an array?

Not wedded to this idea, so just a thought.

Reply:
Done, makes more sense indeed!